packages/flutter_localizations/lib/src/l10n/localizations.dart

```diff
@@ -2,2948 +2,9578 @@
-  @override String get anteMeridiemAbbreviation => r'AM';
-  @override String get postMeridiemAbbreviation => r'PM';
-  @override String get timePickerHourModeAnnouncement => r'Pilih jam';
-  @override String get timePickerMinuteModeAnnouncement => r'Pilih menit';
-  @override String get modalBarrierDismissLabel => r'Tutup';
-  @override String get signedInLabel => r'Telah login';
-  @override String get hideAccountsLabel => r'Sembunyikan akun';
-  @override String get showAccountsLabel => r'Tampilkan akun';
-  @override String get drawerLabel => r'Menu navigasi';
-  @override String get popupMenuLabel => r'Menu pop-up';
-  @override String get dialogLabel => r'Dialog';
-  @override String get alertDialogLabel => r'Waspada';
-  @override String get searchFieldLabel => r'Pencarian';
-}
-
-// ignore: camel_case_types
-class _Bundle_it extends TranslationBundle {
-  const _Bundle_it() : super(null);
-  @override String get scriptCategory => r'English-like';
-  @override String get timeOfDayFormat => r'HH:mm';
-  @override String get selectedRowCountTitleOne => r'1 elemento selezionato';
-  @override String get openAppDrawerTooltip => r'Apri il menu di navigazione';
-  @override String get backButtonTooltip => r'Indietro';
-  @override String get closeButtonTooltip => r'Chiudi';
-  @override String get deleteButtonTooltip => r'Elimina';
-  @override String get nextMonthTooltip => r'Mese successivo';
-  @override String get previousMonthTooltip => r'Mese precedente';
-  @override String get nextPageTooltip => r'Pagina successiva';
-  @override String get previousPageTooltip => r'Pagina precedente';
-  @override String get showMenuTooltip => r'Mostra il menu';
-  @override String get aboutListTileTitle => r'Informazioni su $applicationName';
-  @override String get licensesPageTitle => r'Licenze';
-  @override String get pageRowsInfoTitle => r'$firstRow-$lastRow di $rowCount';
-  @override String get pageRowsInfoTitleApproximate => r'$firstRow-$lastRow di circa $rowCount';
-  @override String get rowsPerPageTitle => r'Righe per pagina:';
-  @override String get tabLabel => r'Scheda $tabIndex di $tabCount';
-  @override String get selectedRowCountTitleOther => r'$selectedRowCount elementi selezionati';
-  @override String get cancelButtonLabel => r'ANNULLA';
-  @override String get closeButtonLabel => r'CHIUDI';
-  @override String get continueButtonLabel => r'CONTINUA';
-  @override String get copyButtonLabel => r'COPIA';
-  @override String get cutButtonLabel => r'TAGLIA';
-  @override String get okButtonLabel => r'OK';
-  @override String get pasteButtonLabel => r'INCOLLA';
-  @override String get selectAllButtonLabel => r'SELEZIONA TUTTO';
-  @override String get viewLicensesButtonLabel => r'VISUALIZZA LICENZE';
-  @override String get anteMeridiemAbbreviation => r'AM';
-  @override String get postMeridiemAbbreviation => r'PM';
-  @override String get timePickerHourModeAnnouncement => r'Seleziona le ore';
-  @over
```

Comment:
I've just realized that this doesn't work for small numbers. By virtue of speaking the language, I know that 'de' is added for `$selectedRowCount >= 20`, while it cannot be there for anything less, but I'm having a hard time finding a grammatical explanation for it. Numbering being something you learn so early in life, it's not a very popular subject in schools. My hypothesis is that numerals which are composed of multiplications of other numerals, e.g. 'douăzeci' which is literally 'two tens' vs. 'nouăsprezece' which is literally 'nine towards ten', need this, which means basically all of them starting from 20 onward.

Reply:
I'm just basing this on the CLDR and what `intl` implements:
http://www.unicode.org/cldr/charts/latest/supplemental/language_plural_rules.html#ro